torch/utils/collect_env.py

```diff
@@ -654,11 +654,12 @@ def get_pip_packages(run_lambda, patterns=None):
     return pip_version, filtered_out
 
 
-def get_cachingallocator_config():
-    ca_config = os.environ.get("PYTORCH_CUDA_ALLOC_CONF", "")
-    if not ca_config:
-        ca_config = os.environ.get("PYTORCH_HIP_ALLOC_CONF", "")
-    return ca_config
+def get_cachingallocator_config() -> str:
```

Comment:
Let's print all 3 actually. I could see it being helpful when users confusingly define more than one

Reply:
Done. The only concern is this function is a public API.